src/Field/Money.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+// vim:ts=4:sw=4:et:fdm=marker:fdl=0
+
+namespace atk4\data\Field;
+
+use atk4\data\Field;
+
+/**
+ * Class Money offers a lightweight implementation of currencies.
+ */
+class Money extends Numeric
+{
+    /** @var string Field type for backward compatibility. */
+    public $type = 'money';
+
+    /**
+     * @var int Specify how many decimal numbers should be saved.
+     */
+    public $decimal_numbers = 4;
```

Comment:
normally that's 2

Reply:
it's 4 currently so I thought we should have the same (for BC), but normally it should be 2 in most cases.

Reply:
addressed